qemu/tests/numa_node_affinity.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+
+from virttest.qemu_monitor import QMPCmdError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    numa_node_affinity test
+    1) Check the NUMA topology, cancel if there is a single node.
+    2) Boot up a guest with node-affinity selecting the first valid node.
+    3) Check the cpu-affinity obtained from QEMU is correct.
+    4) Check the node-affinity property is not readable.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    host_numa_node = utils_misc.NumaInfo()
+    node_list = host_numa_node.online_nodes_withcpu
+    if len(node_list) < 2:
+        test.cancel("Host only has one NUMA node, skipping test...")
+
+    error_msg = params.get("error_msg", "")
```

Comment:
How do you think if we set it like this ?
 error_msg = params.get("error_msg", "Property 'thread-context.node-affinity' is not readable") ?

Reply:
Updated, thanks !